portals/devportal/source/src/app/components/Applications/Listing/AppsTableContent.jsx

```diff
@@ -156,7 +156,7 @@ class AppsTableContent extends Component {
                                             <Typography variant='caption'>
                                                 <FormattedMessage
                                                     id='Applications.Listing.AppsTableContent.wait.approval'
-                                                    defaultMessage='waiting for approval'
```

Comment:
Better to add a style attribute to the <Typography> element before this at [1].

```css
style={{textIndent: '10px'}}
```
[1] - https://github.com/wso2/apim-apps/blob/a3e1c555c6341f27510e5828580c1af50e29ad7c/portals/devportal/source/src/app/components/Applications/Listing/AppsTableContent.jsx#L149